batch/batches/KBatchBase.class.php

```diff
@@ -892,4 +892,16 @@ public static function getConfigParam($key, $mapName = 'local' , $defaultValue =
 		}
 		return $defaultValue;
 	}
+
+	/**
+	 * Calling to listAction on the required $service
+	 * @param $service
+	 * @param $filter
+	 * @param $pager
+	 * @return mixed|null
+	 */
+	public static function apiListCall($service, $filter, $pager)
+	{
+		return self::$kClient->$service->listAction($filter, $pager);
```

Comment:
return self::tryExecuteApiCall(self::$kClient->$service->listAction, array($filter, $pager));

To be use as: 
$serverNodes = self::apiListCall('serverNode', $filter, $pager);